index.html

```diff
@@ -447,6 +447,11 @@
 &lt;/script&gt;
 
 </pre>
+      </section>
+      <section>
+        <h3>
+          Receiver: monitor available session and say hello
```

Comment:
Is "receiver" defined yet in the document?  Would restate as:

Presenting context: monitor available session(s) and say hello.